cmd/docker/docker.go

```diff
@@ -55,7 +55,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	cli.SetupRootCommand(cmd)
 
 	flags = cmd.Flags()
-	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
+	flags.BoolVarP(&opts.Version, "version", "v", false, "Show version information")
```

Comment:
@AkihiroSuda ,yes, many "show xxx" commands actually quit, but not add  "and quit", except “docker -v”.